IDEMPIERE-5932 Update English top-level menu names and window names to better support new user adoption

CarlosRuiz-globalqss opened the pull request:
https://idempiere.atlassian.net/browse/IDEMPIERE-5932

# Pull Request Checklist

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] I have checked my code and corrected any misspellings
- [x] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
### Tests
- [x] I have tested the direct scenario that my code is solving
- [x] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [x] New and existing unit tests pass locally with my changes
- [x] I have added unit tests that prove my fix is effective or that my feature works

hengsin commented on migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql:
```diff
@@ -0,0 +1,206 @@
+;
+
+-- Dec 8, 2023, 6:15:33 AM CST
+UPDATE AD_Menu SET Name='Sales Invoice and Note', Description='Customer Invoice Entry', Updated=TO_TIMESTAMP('2023-12-08 06:15:33','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=178
```
Description is not in sync. Also, perhaps clearer to use **Sales Invoice and Credit/Debit Note** instead ?

hengsin commented on migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql:
```diff
@@ -0,0 +1,206 @@
+;
+
+-- Dec 8, 2023, 6:15:56 AM CST
+UPDATE AD_Menu SET Name='Purchase Invoice and Note', Description='Vendor Invoice Entry', Updated=TO_TIMESTAMP('2023-12-08 06:15:56','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=206
```
Description is not in sync. Also, perhaps clearer to use Purchase Invoice and Credit/Debit Note instead ?

hengsin commented on migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql:
```diff
@@ -0,0 +1,206 @@
+;
+
+-- Dec 8, 2023, 6:12:46 AM CST
+UPDATE AD_Menu SET Name='Inventory Decrease/Increase', Description='Enter Internal Use of Inventory', Updated=TO_TIMESTAMP('2023-12-08 06:12:46','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=503
```
Description is not in sync.

hengsin commented on migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql:
```diff
@@ -0,0 +1,206 @@
+;
+
+-- Dec 8, 2023, 6:08:21 AM CST
+UPDATE AD_Menu SET Name='Open Items (Aging)',Updated=TO_TIMESTAMP('2023-12-08 06:08:21','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=243
```
The report title is **Open Item (Invoice) List**, perhaps the two should be in sync ?

CarlosRuiz-globalqss commented:
Ready @hengsin - I implemented your suggestions.

Testing this I also noticed that favourites were showing the name as tooltip, while the menu is showing the description as tooltip, so I fixed the favourite to show description too.

hengsin reviewed (approved):
looks good
